strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout-provider/src/main/java/org/carlspring/strongbox/artifact/MavenRepositoryArtifact.java

```diff
@@ -42,22 +51,22 @@ public MavenDetachedArtifact(final String groupId,
         super(groupId, artifactId, version, type, classifier);
     }
 
-    public MavenDetachedArtifact(final String groupId,
+    public MavenRepositoryArtifact(final String groupId,
                                  final String artifactId,
                                  final String version,
                                  final String type)
     {
         super(groupId, artifactId, version, type);
     }
 
-    public MavenDetachedArtifact(final String groupId,
+    public MavenRepositoryArtifact(final String groupId,
                                  final String artifactId,
                                  final String version)
     {
         super(groupId, artifactId, version);
     }
 
-    public MavenDetachedArtifact(final String groupId,
+    public MavenRepositoryArtifact(final String groupId,
```

Comment:
So, what is a `MavenRepositoryArtifact` and how is it different to the `MavenDetachedArtifact`...?